client/src/featureform/serving.py

```diff
@@ -1020,6 +1036,60 @@ def dataframe(self) -> pd.DataFrame:
             self._dataframe.rename(columns={cols.label: "label"}, inplace=True)
             return self._dataframe
 
+    def _get_file_format(self, location: str) -> str:
```

Comment:
I would argue that you can put this function in the FileFormat class where you can pass in it a default